File: prime.py
#! /usr/bin/env python2

import math
def isPrime(n):
    if n%2==0:
        return False
    else:
        for e in range(3,int(math.sqrt(n)+1)):
            if n%e==0:
                return False
        return True

for i in range(1,100):
    if isPrime(i):
        print (i)

File: prime.js

function isPrime(n){
	if(n%2==0) return false;
	for(var i=3;i<Math.sqrt(n)+1;i++){
		if(n%i==0) return false;
	}
	return true;
}

for(var i=1;i<100;i++){
	if(isPrime(i)) console.log(i);
}
